test/lift.js

```diff
@@ -40,4 +41,10 @@ describe('lift', function() {
     it('throws on zero arguments', function() {
         assert.throws(R.lift);
     });
+
+    it('works with other functors such as "Maybe"', function() {
+        var addM = R.lift(R.add);
+        assert.deepEqual(addM(Maybe(3), Maybe(5)), Maybe(8));
+    });
```

Comment:
While I don't mind switching to Id, my thought was to be able to use this for multiple tests across many different functions, so it isn't clear to me what functions are really necessary.  Even here, for the current implementation of `lift` we need `ap` and `map`.  But it might well be reasonable to consider reimplementing it in terms of `of` and `ap`.  I wouldn't want to tie our tests too closely to our current implementation.